File: gameData.js
class Team {
  constructor(name, score) {
    this.name = name;
    this.score = score;
  }
}

class GameData {
  constructor(team1, team2, mode) {
    this.team1 = team1;
    this.team2 = team2;
    this.mode = mode;
  }
}

function stringToGameData(string) {
  ret = JSON.parse(string)["GameData"];
  ret.team1 = ret.team1["Team"];
  ret.team2 = ret.team2["Team"];

  return ret;
}

File: gameData.py
import enum
import json

class Mode(enum.Enum):
    PAUSE=1
    LETTERS=2
    NUMBERS=2

class _Team():
    def __init__(self, number):
        self.name = "Team{0}".format(number)
        self.score = 0

    def toJSON(self):
        return json.dumps({"Team": {
            "name": self.name,
            "score": self.score
        }})

# TODO read data from saved file
class GameData():
    def __init__(self, fileName=None):
        self.team1 = _Team(1)
        self.team2 = _Team(2)

        self.mode = Mode.PAUSE

    def toJSON(self):
        return json.dumps({"GameData": {
            "team1": json.loads(self.team1.toJSON()),
            "team2": json.loads(self.team2.toJSON()),
            "mode": str(self.mode)
        }})
